### PR TITLE
Fix lint

### PR DESCRIPTION
**Pull request by Jeymisson**
- Fixed lint errors
- Add husky to prevent commits with lint errors

### CODE REVIEW

File: react/components/ErrorBoundary.tsx [hunk 1]
```diff
@@ -7,32 +7,33 @@ interface Props {
...
 
 class ErrorBoundary extends React.Component<Props> {
-  state = {
+  private _state = {
     error: undefined,
     errorInfo: undefined,
   }
 
-  componentDidCatch(error: Error, errorInfo: ErrorInfo) {
+  public get state() {
```
Comment (thiagomurakami) [inline react/components/ErrorBoundary.tsx]: Since it's React API, maybe it isn't a good idea to shouldn't override `this.state`

File: react/components/ErrorBoundary.tsx [hunk 2]
```diff
@@ -7,32 +7,33 @@ interface Props {
 }
 
 class ErrorBoundary extends React.Component<Props> {
-  state = {
+  private _state = {
```
Comment (thiagomurakami) [inline react/components/ErrorBoundary.tsx]: Maybe `public state`? Why did you need to use `private _state`?

Comment (Jeymisson) [reply react/components/ErrorBoundary.tsx]: It was a vscode suggestion, but believe they're the same thing.

Comment (Jeymisson) [reply react/components/ErrorBoundary.tsx]: Will leave with the pattern we already use though.
